cmd/imgcollector/image/hostfs/containerd_image.go

```diff
@@ -42,6 +42,11 @@ func NewContainerdImage(hash v1.Hash, cfg ContainerdHostFSConfig) (Image, error)
 		configBytes: configBytes,
 		contentDir:  cfg.ContentDir,
 	}
+
+	index, err := manifestReader.resolveIndex()
+	if err == nil {
```

Comment:
In this way we also lose information on any other errors. what about creating new error for index missing, and log the error if it's not it?